traversal/selector/parse/selector_parse.go

```diff
@@ -67,13 +67,13 @@ func must(s selector.Selector, e error) selector.Selector {
 // It doesn't walk anywhere at all.
 //
 // This is not a very useful selector, but is an example of how selectors can be written.
-var CommonSelector_MatchPoint = must(ParseAndCompileJSONSelector(`{".":{}}`))
+var CommonSelector_MatchPoint = must(ParseJSONSelector(`{".":{}}`))
```

Comment:
I'd really like these to not be in the `parse` package at all. see #249 
Needed to point other people to them rather than re-implementing from a builder, and it took way too long for either of us to find where they'd been defined.

Reply:
I'm not thrilled with it either but the fix for both {make things discoverable from the more root-wards package} plus {selectors are possible to use without a strict dependency on dag-json} is going full ham on https://github.com/ipld/go-ipld-prime/issues/236 (which... might be a one-day effort, actually, come to think of it.  But might be more than that by the time we talk about change propagation).

Reply:
Yeah, I just figured that instead of me attempting to pull off #236 I'd just do a piece of it that's currently getting in my way. If it would help that effort to move them as part of this PR then I'm happy to change.